src/vs/workbench/browser/quickopen.ts

```diff
@@ -250,35 +256,53 @@ export class EditorQuickOpenEntry extends QuickOpenEntry implements IEditorQuick
 	public run(mode: Mode, context: IEntryRunContext): boolean {
 		const hideWidget = (mode === Mode.OPEN);
 
+		let sideBySide;
 		if (mode === Mode.OPEN || mode === Mode.OPEN_IN_BACKGROUND) {
-			let sideBySide = context.keymods.indexOf(KeyMod.CtrlCmd) >= 0;
-
-			let openInBackgroundOptions: IEditorOptions;
-			if (mode === Mode.OPEN_IN_BACKGROUND) {
-				openInBackgroundOptions = { pinned: true, preserveFocus: true };
-			}
+			sideBySide = !context.quickNavigateConfiguration && context.keymods.indexOf(KeyMod.CtrlCmd) >= 0;
+		}
 
-			let input = this.getInput();
-			if (input instanceof EditorInput) {
-				let opts = this.getOptions();
-				if (opts) {
-					opts = objects.mixin(opts, openInBackgroundOptions, true);
-				} else if (openInBackgroundOptions) {
-					opts = EditorOptions.create(openInBackgroundOptions);
-				}
+		let pinned;
```

Comment:
Im a bit confused by your comment.  Also, I don't see where the editorPart is handling this.  It used to be in the controller, but was moved when I refactored that out here.

It's used in `OPEN_IN_BACKGROUND` and `OPEN` modes and gets it's value from the `workbench.editor.enablePreviewFromQuickOpen` setting while in `OPEN` mode.  Some of this logic was taken when I removed this logic from the controller. 

Did you look at this in context of the whole `run` method here?